code/modules/admin/admin_verbs.dm

```diff
@@ -689,11 +689,11 @@ var/list/admin_verbs_ticket = list(
 	if(!check_rights(R_DEBUG))
 		return
 
-	if(air_processing_killed)

```

Comment:
I'll be completely honest, I'm not even sure if this entire proc should exist anymore.

This harkens back to the days of ZAS, where it lagged like hell and caused lots of problem and was prone to breaking (modern ZAS is much better).

Reply:
You want me to gut that?

Reply:
Done